src/core/Util.js

```diff
@@ -126,6 +126,21 @@ module.exports = {
     var wysiwygDivObserver = new MutationObserver(handleChange);
 
     wysiwygDivObserver.observe(wysiwygDiv, observerConfig);
+  },
+
+  hideFooterWhenTypingOnMobile: function() {
+    var inputArea = $('input, textarea, .wk-wysiwyg');
+
+    if(window.innerWidth <= 992) {
```

Comment:
I think this `if  condition` needs to be inside `focusin` listener?
For example, i am using a tablet and the browser width is less than 992. Now I have maximized the browser such that it width is greater than 992, in that case I donot want the bar to be hidden, right?

Reply:
will change this.